aptos-move/framework/src/extended_checks.rs

```diff
@@ -428,6 +435,87 @@ impl<'a> ExtendedChecker<'a> {
     }
 }
 
+// ----------------------------------------------------------------------------------
+// Events
+
+impl<'a> ExtendedChecker<'a> {
+    fn check_and_record_events(&mut self, module: &ModuleEnv) {
```

Comment:
Does this run for scripts at all? I wonder how we can disable emit event calls for scripts.

Reply:
maybe not. I think we are going to solve the issue.